.travis.yml

```diff
@@ -73,6 +73,7 @@ before_install:
 before_script:
   - docker exec -t devenv /bin/sh -c "cd /opt/shogun/build; cmake -DCMAKE_INSTALL_PREFIX=$HOME/shogun-build -DENABLE_TESTING=ON $CMAKE_OPTIONS .."
 script:
+  - docker exec -t devenv /bin/sh -c "cd /opt/shogun/scripts; bash ./check_format.sh"
```

Comment:
just remove `bash` and make sure that `check_format.sh` is executable and let the `env` find out where bash is ... if available.